src/app/games/quebec-castles/QuebecCastlesTutorial.ts

```diff
@@ -0,0 +1,111 @@
+            QuebecCastlesRules.get().getInitialState(defaultConfig).incrementTurn(),
+            QuebecCastlesMove.translation(new Coord(2, 2), new Coord(4, 4)),
+            TutorialStepMessage.CONGRATULATIONS_YOU_KNOW_EVERYTHING(),
+        ),
+        // 5.a configurations alternatives: rectangulaire & width & height
+        TutorialStep.informational(
+            $localize`Custom config: rhombic, width, height`,
+            $localize`You have the option to change the shape of the board, if it's not rhombic, it's rectangular. You also can change the size`,
+            QuebecCastlesRules.get().getInitialState(rectangularWidthHeightConfig),
+            rectangularWidthHeightConfig,
+        ),
+        TutorialStep.anyMove(
+            $localize`Custom config: place castle yourself`,
+            $localize`You have the option to change decide yourself where you place the castle. If you don't change anything else the piece placement will be automatically done right after.<br/><br/>You're playing Dark/Defender, place your castle.`,
+            QuebecCastlesRules.get().getInitialState(playersPlaceCastleConfig),
+            QuebecCastlesMove.drop([new Coord(7, 7)]),
+            TutorialStepMessage.CONGRATULATIONS(),
+            playersPlaceCastleConfig,
+        ),
+        TutorialStep.anyMove(
+            $localize`Custom config: drop mode: by batch`,
+            $localize`You have the option to decide yourself where you place your pieces.`,
+            QuebecCastlesRules.get().getInitialState(dropByBatchConfig),
+            QuebecCastlesMove.drop([
+                new Coord(4, 8),
+                new Coord(5, 7),
+                new Coord(6, 6),
+                new Coord(7, 5),
+                new Coord(8, 4),
+                new Coord(5, 8),
+                new Coord(6, 7),
+                new Coord(7, 6),
+                new Coord(8, 5),
+            ]),
+            TutorialStepMessage.CONGRATULATIONS(),
+            dropByBatchConfig,
+        ),
+        TutorialStep.anyMove(
+            $localize`Custom config: Drop Mode: Piece by piece`,
+            $localize`You have the option to decide yourself where you place your pieces, one per turn. Once one player has no more piece to drop, the other player drop all its piece in one turn.<br/><br/>You're playing Dark/Defender, drop a piece.`,
+            QuebecCastlesRules.get().getInitialState(dropPieceByPieceConfig),
+            QuebecCastlesMove.drop([new Coord(7, 7)]),
+            TutorialStepMessage.CONGRATULATIONS(),
+            dropPieceByPieceConfig,
+        ),
+        TutorialStep.informational(
+            $localize`Custom config: Number of piece & territory size`,
+            $localize`You have the option to change territory size, which is the number of line on which you can drop your pieces, it's indicated by a semi-transparent color. You can also change how many pieces each player control.`,
```

Comment:
the number of lines*
mettre un point avant "it's", et changer "it's" en "It is"